lib/smart_proxy_pulp_plugin/disk_usage.rb

```diff
@@ -0,0 +1,39 @@
+module PulpProxy
+  class DiskUsage
+    require 'sys/filesystem'
+    include Sys
+
+    SIZE = {:kilobyte => 1_024, :megabyte => 1_048_576, :gigabyte => 1_073_741_824, :terabyte => 1_099_511_627_776}
+
+    attr_reader :path, :stat, :size
+
+    def initialize(path = PulpProxy::Plugin.settings.pulp_dir, size = SIZE[:megabyte])
+      @path = path
```

Comment:
I would pass opts = {} instead of hard coding the size as the second argument, e.g.
```ruby
DiskUsage.new(:path => '..', :size_format ..)
```